### PR TITLE
improve(api): make api a public package

### PR DESCRIPTION
**Pull request by daywiss**
Signed-off-by: David Adams <david@umaproject.org>

**Motivation**

https://app.shortcut.com/uma-project/story/2451/prepare-api-to-run-as-library

**Summary**

Exposes code through out the api, makes it public to allow publishing.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

https://app.shortcut.com/uma-project/story/2451/prepare-api-to-run-as-library

### CODE REVIEW

File: packages/api/src/index.ts [hunk 1]
```diff
@@ -1,120 +1,6 @@
-import uma from "@uma/sdk";
```
Comment (mrice32) [inline packages/api/src/index.ts:1]: I'm assuming all this was just moved?

**Review by mrice32 — APPROVED** [review]
@daywiss is this for the automated testing framework?

**Comment by daywiss** [comment]
> @daywiss is this for the automated testing framework?

Yea exactly

File: packages/api/src/index.ts (hunk 1 above)
Comment (daywiss) [reply packages/api/src/index.ts:1]: Yes moved to the types file. Maybe it should be import type

**Review by amateima — APPROVED** [review]

**Pull request merged**